### PR TITLE
[Compute] Add VmssVM PUT API.

### PR DESCRIPTION
**Pull request by hyonholee**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/2017-12-01/compute.json [hunk 1]
```diff
@@ -5897,6 +5951,18 @@
         "licenseType": {
           "type": "string",
           "description": "Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br><br> Possible values are: <br><br> Windows_Client <br><br> Windows_Server <br><br> If this element is included in a request for an update, the value must match the initial value. This value cannot be updated. <br><br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-hybrid-use-benefit-licensing?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) <br><br> Minimum api-version: 2015-06-15"
+        },
+        "priority": {
```
Comment (anuchandy) [inline specification/compute/resource-manager/Microsoft.Compute/2017-12-01/compute.json]: Addition of settable properties to an existing model (`VirtualMachineScaleSetVMProfile`) of an existing api version is considered as breaking change. 

It looks like this model `VirtualMachineScaleSetVMProfile` is used in `VirtualMachineScaleSet` model [VirtualMachineScaleSet -> VirtualMachineScaleSetProperties -> VirtualMachineScaleSetVMProfile] which represent payload of already supported VMSS create or update op. that means we are adding settable properties to VMSS create_update payload.

Adding ARM to review this.

**Comment by anuchandy** [comment]
@ravbhatnagar please take a look

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json [hunk 2]
```diff
@@ -3083,6 +3083,60 @@
...
+            "description": "Parameters supplied to the Update Virtual Machine Scale Sets VM operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```
Comment (ravbhatnagar) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: Add the final 200 OK response also under "responses".

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json [hunk 3]
```diff
@@ -5897,6 +5951,18 @@
         "licenseType": {
           "type": "string",
           "description": "Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br><br> Possible values are: <br><br> Windows_Client <br><br> Windows_Server <br><br> If this element is included in a request for an update, the value must match the initial value. This value cannot be updated. <br><br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-hybrid-use-benefit-licensing?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) <br><br> Minimum api-version: 2015-06-15"
+        },
+        "priority": {
```
Comment (ravbhatnagar) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: This requires an API-version change as a new writable property is getting added.

**Comment by anuchandy** [comment]
@hyonholee what's the plan here? are we going to bump up the api version based on Gaurav's comment?

**Comment by hyonholee** [comment]
@anuchandy  I removed the priority part, which Gaurav said requires API version update.  Now this PR only contains VMSSVM PUT operation.  This is additional operation and does not modify existing API, so does not require API version update.

**Comment by hyonholee** [comment]
@anuchandy For priority, I will send a separate PR after we resolve the issue.

**Review by anuchandy — APPROVED** [review]
@ravbhatnagar PR looks good to me, can you please sign-off?

**Comment by ravbhatnagar** [comment]
Signing off!

**Comment by hyonholee** [comment]
@anuchandy Please merge this PR.  I will send a separate PR for priority right after this PR is merged.  Thanks.

**Comment by anuchandy** [comment]
Thanks @ravbhatnagar.  @hyonholee merging.

**Pull request merged**